src/features/offer/components/OfferBody/OfferBody.tsx

```diff
@@ -354,6 +359,17 @@ export const OfferBody: FunctionComponent<Props> = ({
         </AccordionItem>
       </SectionWithDivider>
 
+      {!!shouldDisplaySameArtistPlaylist && (
```

Comment:
J'ai vu passer [cet article](https://medium.com/@Evelyn.Taylor/its-2023-please-stop-using-for-conditional-rendering-in-react-be6cbcd72cba), je pense que je vais le considérer à l'avenir mais à toi de voir

Reply:
C'est pas cool le `&&`. Dans ce cas ce n'est pas trop visible mais en interne le résultat de cette opération peut être `false`. Heureusement dans ce cas React sait qu'il ne doit pas afficher `false` mais c'est quand même pas cool, il veut mieux retourner `null`